### PR TITLE
Fix watermarkstat CLI script for unconfigured PG counters

### PR DESCRIPTION
**Pull request by shlomibitton**
Signed-off-by: Shlomi Bitton <shlomibi@nvidia.com>

#### What I did

Since PG counters are created only if they are configured in the switch, it is not enough to relay only on the first entry in the DB when building the output table of watermarkstat script.
We need to go over all configured counters, check what is the max configured, and build the table accordingly.

#### How I did it

Iterate all configured PG buffers for all ports and find the max index.
Build the output table according to the max index.

#### How to verify it

Run test "iface_namingmode/test_iface_namingmode.py" including this PR: https://github.com/Azure/sonic-swss/pull/2143 and observe it passes.

### CODE REVIEW

File: scripts/watermarkstat [hunk 1]
```diff
@@ -208,16 +208,16 @@ class Watermarkstat(object):
...
-        header_len = len(header_map[single_key])
-        min_idx = sys.maxsize
-
-        for name, counter_oid in header_map[single_key].items():
-            curr_idx = int(wm_type["idx_func"](counter_oid))
-            min_idx = min(min_idx, curr_idx)
-
-        self.min_idx = min_idx
-        self.header_list += ["{}{}".format(wm_type["header_prefix"], idx) for idx in range(self.min_idx, self.min_idx + header_len)]
+	self.min_idx = 0
```
Comment (nazariig) [inline scripts/watermarkstat:211]: @shlomibitton why not to use `tabulate` here?

Comment (shlomibitton) [reply scripts/watermarkstat:211]: the tabulate is used at the end of this table creation.
this function build only the table header which later on we add the needed information.
please check this:
https://github.com/shlomibitton/sonic-utilities/blob/409da5ff87cb8864cd41c7b23e62ad4849ff5037/scripts/watermarkstat#L268

File: scripts/watermarkstat [hunk 2]
```diff
@@ -208,16 +208,16 @@ class Watermarkstat(object):
...
         self.header_list = ['Port']
         header_map = wm_type["obj_map"]
-        single_key = list(header_map.keys())[0]
-        header_len = len(header_map[single_key])
-        min_idx = sys.maxsize
 
-        for name, counter_oid in header_map[single_key].items():
-            curr_idx = int(wm_type["idx_func"](counter_oid))
-            min_idx = min(min_idx, curr_idx)
+        self.min_idx = 0
```
[the remark this answers is not in the document]
Comment (shlomibitton) [reply scripts/watermarkstat:212]: it was like this before, not sure if it is used on other places and why the author decaled it with a 'this' so I left it untouched.

[the remark this answers is not in the document]
Comment (nazariig) [reply scripts/watermarkstat:212]: @shlomibitton are u sure? From the code i can see:
```
min_idx = sys.maxsize
```

File: scripts/watermarkstat (hunk 1 above)
Comment (nazariig) [reply scripts/watermarkstat]: @shlomibitton ok

File: scripts/watermarkstat (hunk 2 above)
[the remark this answers is not in the document]
Comment (nazariig) [reply scripts/watermarkstat:212]: @shlomibitton ok. Let's have it as is.

**Pull request closed without merging**